libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -263,6 +263,30 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     keywords: ["avax", "avalanche"],
   },
+  avalanchepchain: {
+    type: "CryptoCurrency",
+    id: "avalanchepchain",
+    coinType: 9000,
+    name: "Avalanche (P-Chain)",
+    managerAppName: "Avalanche",
+    ticker: "AVAX",
```

Comment:
@gre Can it be an issue that two currencies have the same ticker? :thinking:

Reply:
We could change the P-Chain ticker to `AVAX (P-Chain)`. What do you think?

Reply:
I think that solves the potential conflict issues, I'm just wondering if it may cause display issues as tickers are usually shorter.
But it should be fine you can use that value, in any case we can change the ticker later. Only the `id` can never change once it's been released.

Reply:
Follow-up on this question: @trentkrogers just to let you know about our ongoing C-chain implementation going on [here](https://github.com/LedgerHQ/ledger-live/pull/1712).